website/source/docs/vault-integration/index.html.md

```diff
@@ -236,9 +236,11 @@ token_renewable true
 token_policies  [default nomad-server]
 ```
 
-`-orphan` is included above to prevent revocation of the token when its parent expires. 
-See the [Vault token hierarchy](https://www.vaultproject.io/docs/concepts/tokens.html#token-hierarchies-and-orphan-tokens) 
-documentation for more information.
+The `-orphan` flag is included when generating the Nomad server token above to prevent revocation of the token when its parent expires. Vault typically creates tokens with a parent-child relationship.  When an ancestor token is revoked, all of its descendant tokens—and all of their leases—are revoked as well.  
```

Comment:
Change:
When an ancestor token is revoked, all of its descendant tokens—and all of their leases—are revoked as well.
To
When an ancestor token is revoked, all of its descendant tokens and their associated leases are revoked as well.